src/server/config.rs

```diff
@@ -28,6 +28,9 @@ const DEFAULT_RECV_BUFFER_SIZE: usize = 128 * 1024;
 pub struct Config {
     pub cluster_id: u64,
 
+    // Server tags.
```

Comment:
A comment should be helpful, which explains how to use tags to achieve replica constraints and how to set these tags correctly.